downstream/assemblies/eda/assembly-eda-projects.adoc

```diff
@@ -6,6 +6,12 @@ Projects are a logical collection of rulebooks.
 They must be a git repository and only http protocol is supported. 
 The rulebooks of a project must be located in the path defined for {EDAName} content in Ansible collections: `/extensions/eda/rulebooks` at the root of the project.
 
+[NOTE]
+====
+* Do not link a project to an empty repository. Linking a project to an empty repository causes synchronization errors.
+* When Redis is unavailable due to a failure, you will not be able to create or sync projects.
```

Comment:
I'm not certain about the inclusion of "due to a failure."  It's fairly generic and doesn't really add anything, imo.
Perhaps something along the line of "If EDA cannot successfully communicate with Redis..." ?  But I might be acting too "engineery".

Reply:
I can see your point with that @jshimkus-rh. I followed some suggestions, but will remove that portion. Thanks for feedback on that.